src/pipe_wrap.cc

```diff
@@ -98,7 +77,8 @@ void PipeWrap::Initialize(Local<Object> target,
 
   // Create FunctionTemplate for PipeConnectWrap.
   Local<FunctionTemplate> cwt =
-      FunctionTemplate::New(env->isolate(), NewPipeConnectWrap);
+      FunctionTemplate::New(env->isolate(),
+      [](const FunctionCallbackInfo<Value>& a){CHECK(a.IsConstructCall());});
```

Comment:
That looks much better...I was desperately trying to squeeze that into a single line.